app/stac_api/views.py

```diff
@@ -106,6 +112,92 @@ def parse_datetime_query(date_time):
     return start, end
 
 
+def filter_by_bbox(queryset, bbox):
+    '''Filter a querystring with a given bbox
+
+    This function is a helper function, for some views, to add a bbox filter to the queryset.
```

Comment:
FOR NEXT PR: following the object-oriented approach of django, this would be a candidate for creating custom queryset class: https://docs.djangoproject.com/en/3.1/topics/db/managers/#calling-custom-queryset-methods-from-the-manager

Reply:
https://github.com/geoadmin/service-stac/pull/175